### PR TITLE
Add command to update status

### PR DESCRIPTION
**Pull request by ifosch**
This PR starts Mallerenga development.
It contains the first MVP, which is a command to have the Twitter status updated.

### CODE REVIEW

File: mallerenga/twitter/twitter.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+class Twitter(object):
+    def __init__(self):
+        if validate_credentials():
+            consumer_key = os.environ['TWITTER_CONSUMER_KEY']
+            consumer_secret = os.environ['TWITTER_CONSUMER_SECRET']
+            access_token = os.environ['TWITTER_ACCESS_TOKEN']
+            access_token_secret = os.environ['TWITTER_ACCESS_TOKEN_SECRET']
+        else:
+            raise Exception(
+                'Invalid or incomplete Twitter credentials in envorinment',
```
Comment (DZPM) [inline mallerenga/twitter/twitter.py]:
```suggestion
                'Invalid or incomplete Twitter credentials in environment',
```

File: test/mallerenga/twitter/test_twitter.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+from mallerenga.twitter.twitter import Twitter
+
+
+@pytest.mark.parametrize(
+    'environ,exception_message',
+    [
+        (
+            'missing',
+            'Invalid or incomplete Twitter credentials in envorinment',
```
Comment (DZPM) [inline test/mallerenga/twitter/test_twitter.py]:
```suggestion
            'Invalid or incomplete Twitter credentials in environment',
```

File: test/mallerenga/twitter/test_twitter.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        ),
+        (
+            'incomplete',
+            'Invalid or incomplete Twitter credentials in envorinment',
```
Comment (DZPM) [inline test/mallerenga/twitter/test_twitter.py]:
```suggestion
            'Invalid or incomplete Twitter credentials in environment',
```

File: test/mallerenga/twitter/test_twitter.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        ),
+        (
+            'empty',
+            'Invalid or incomplete Twitter credentials in envorinment',
```
Comment (DZPM) [inline test/mallerenga/twitter/test_twitter.py]:
```suggestion
            'Invalid or incomplete Twitter credentials in environment',
```

**Review by DZPM — CHANGES_REQUESTED** [review]

**Comment by ifosch** [comment]
@DZPM I fixed the typo

**Review by DZPM — APPROVED** [review]

**Pull request merged**